src/lib.rs

```diff
@@ -263,7 +263,7 @@
 //! ## Who's using SeaORM?
 //!
 //! SeaORM is the foundation of [StarfishQL](https://github.com/SeaQL/starfish-ql), an experimental graph database and query engine.
-//! 
+//!
```

Comment:
This was changed after running `cargo fmt`

Reply:
Nice!